cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -133,7 +133,7 @@ function start_addon() {
   local -r delay=$3;
   local -r namespace=$4
 
-  create_resource_from_string "$(cat ${addon_filename})" "${tries}" "${delay}" "${addon_filename}" "${namespace}"
+  create_resource_from_string "$(cat "${addon_filename}")" "${tries}" "${delay}" "${addon_filename}" "${namespace}"
```

Comment:
Do these `"` need to be `"\""`?

Reply:
no, this parses as `cat "${addon_filename}"` being up to the subshell